spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -855,18 +874,17 @@ Message<O> method(Message<I> msg)
 O method(I payload)
 ----
 | Post-Processing
-| None, Pre-Processing, Post-Processing
+| None, Pre-Processing, Post-Processing (when the message wrapping the produced payload is acknowledged)
 
 |
 [source,java]
 ----
 @Incoming("in")
 @Outgoing("out") 
-CompletionStage<Message<O>> 
-method(Message<I> msg)
+CompletionStage<Message<O>> method(Message<I> msg)
 ----
-| Post-Processing
-| None, Manual, Pre-Processing, Post-Processing
+| Pre-Processing
```

Comment:
Cross check against "When dealing with single payloads, the POST_PROCESSING strategy is the default strategy."

Reply:
Fine  not a payload